acestep/training_v2/ui/summary.py

```
@@ -37,38 +38,76 @@ def show_summary(
     """
+    if stats.current_step == 0 and stats.current_epoch == 0:
+        _show_failure(stats, output_dir)
+        return
+
     if is_rich_active() and console is not None:
         _show_rich(stats, output_dir, log_dir)
     else:
         _show_plain(stats, output_dir, log_dir)
 
 
+def _show_failure(stats: TrainingStats, output_dir: str) -> None:
```

Ruff (0.15.0)

[warning] Unused function argument: `stats`

(ARG001)